tests/test_modeling_common.py

```diff
@@ -1327,16 +1328,30 @@ def _create_and_check_torch_fx_tracing(self, config, inputs_dict, output_loss=Fa
                             (past_mask, inputs_to_test[1]["attention_mask"]), dim=1
                         )
 
+            if "inputs_embeds" in inspect.signature(model.forward).parameters:
+                inputs_to_test.append(
+                    {
+                        "inputs_embeds": torch.rand(
+                            3, 5, model.config.hidden_size, dtype=torch.float, device=torch_device
+                        )
+                    }
+                )
```

Comment:
shouldn't this be handled by `_prepare_for_class`? Especially as it would avoid this hard-coded 3, 5

Reply:
Happy to, but `inputs_embeds` does not seem to be tested (AFAIK it is not in general in `prepare_config_and_inputs_for_common`). Do you suggest to edit all the `tests/models/**/test_modeling_*.py` to get `inputs_embeds`?